README.md

```diff
@@ -215,6 +215,8 @@ Table of Contents
   * https://mailstache.io/ - 4 Mailboxes @ 1GB each for up to 2 custom domains.
   * https://postmarkapp.com/ - First 25,000 emails are free
   * https://www.zoho.com/mail/ - Free Email management and collaboration for up to 10 users.
+  * https://pdd.yandex.ru - Free email&dns hosting for up to 1000 users.
+  * https://biz.mail.ru/ - Free email hosting.
```

Comment:
I am sure these services are good and fit the bill but without the pages being in english I can't verify it and so have not been accepting entries from services that's not in English :(

I have to maintain the list in the long term and simply cant understand them then I can't, happy to accept the fogcreek one though but would need the non english ones remove - or updated with english URLs if you have some